edison/templates/home.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'layout.html' %}
+
+{% block body %}
+{% include 'include/footer.html' %}
```

Comment:
shouldn't this go in the layout file as well?

Reply:
I think that some pages will be with footer and others not, this is why I put it there

Reply:
Not sure why we would want pages not to have a footer. We could have another layout page that inherits the main one and adds a footer so that pages that need a footer will inherit from it.

But we can also start like this and change things later when we have more pages and we're smarter about what we really need.